### PR TITLE
Eth2 showcase with log poller

### PR DESCRIPTION
**Pull request by Tofel**
Shows how to use new proposed Ethereum Network Builder using a log poller test. And how to make it backward compatible with existing tests that just use `Geth` with `PoW`.

### CODE REVIEW

File: integration-tests/universal/log_poller/helpers.go [hunk 1]
```diff
@@ -1068,9 +1068,20 @@ func setupLogPollerTestDocker(
...
+		WithBeaconChainConfig(ctf_test_env.BeaconChainConfig{
+			SecondsPerSlot: 4,
+			SlotsPerEpoch:  2,
+		}).
+		Build()
+
 	env, err = test_env.NewCLTestEnvBuilder().
 		WithTestLogger(t).
-		WithGeth().
+		WithPrivateEthereumNetwork(ethBuilder).
```
Comment (lukaszcl) [inline integration-tests/universal/log_poller/helpers.go:1084]: @Tofel could you please check my comment [here](https://github.com/smartcontractkit/chainlink-testing-framework/pull/764/files#r1389082129)? Why not pass an `EthereumNetwork` object (can be an interface) here to `WithEthereumNetwork(ethNetwork)`?

Comment (Tofel) [reply integration-tests/universal/log_poller/helpers.go]: done

File: integration-tests/docker/test_env/test_env_config.go [hunk 2]
```diff
@@ -3,14 +3,16 @@ package test_env
...
+	cte "github.com/smartcontractkit/chainlink-testing-framework/docker/test_env"
 	env "github.com/smartcontractkit/chainlink/integration-tests/types/envcommon"
 )
 
 type TestEnvConfig struct {
-	Networks    []string          `json:"networks"`
-	Geth        GethConfig        `json:"geth"`
-	MockAdapter MockAdapterConfig `json:"mock_adapter"`
-	ClCluster   *ClCluster        `json:"clCluster"`
+	Networks              []string                   `json:"networks"`
```
[the remark this answers is not in the document]
Comment (Tofel) [reply integration-tests/docker/test_env/test_env_config.go]: we should! but I think we need to move the CTF config first and then we can move product-specific test configs?

File: integration-tests/docker/test_env/test_env_builder.go [hunk 3]
```diff
@@ -311,8 +326,8 @@ func (b *CLTestEnvBuilder) Build() (*CLClusterTestEnv, error) {
 			var httpUrls []string
 			var wsUrls []string
 			if networkConfig.Simulated {
-				httpUrls = []string{internalDockerUrls.HttpUrl}
-				wsUrls = []string{internalDockerUrls.WsUrl}
+				httpUrls = rpcProvider.PrivateHttpUrls()
+				wsUrls = rpcProvider.PrivateWsUrsl()
```
[the remark this answers is not in the document]
Comment (Tofel) [reply integration-tests/docker/test_env/test_env_builder.go:336]: thx! fixed here: https://github.com/smartcontractkit/chainlink-testing-framework/pull/767

**Review by skudasov — APPROVED** [review]

**Pull request merged**